src/main/java/org/apache/commons/lang3/EnumUtils.java

```diff
@@ -67,7 +67,7 @@
     private static <E extends Enum<E>> Class<E> checkBitVectorable(final Class<E> enumClass) {
         final E[] constants = asEnum(enumClass).getEnumConstants();
         Validate.isTrue(constants.length <= Long.SIZE, CANNOT_STORE_S_S_VALUES_IN_S_BITS,
-            Integer.valueOf(constants.length), enumClass.getSimpleName(), Integer.valueOf(Long.SIZE));
+            Integer.valueOf(constants.length), ClassUtils.getSimpleName(enumClass), Integer.valueOf(Long.SIZE));
```

Comment:
This does not make sense since a null enumClass will cause an exception before we even get here.